pymodbus/repl/server/main.py

```diff
@@ -166,6 +166,13 @@ def run(
         help="Randomize every `r` reads. 0=never, 1=always,2=every-second-read"
         ", and so on. Applicable IR and DI.",
     ),
+    change_rate: int = typer.Option(
+        0,
+        "--change-rate",
+        "-c",
+        help="Chance of randomizing reads. 0=never, 100=always, 1=1%"
```

Comment:
```suggestion
        help="Rate in % of registers to change. 0=none, 100=all, 12=12% of registers"
```
I believe this is a better help.

Reply:
IMHO, `--change-rate` ,`--probability` or simply `--likely` are all OK.